.github/workflows/awesome_workflow.yml

```diff
@@ -23,10 +23,9 @@ jobs:
           git config --global user.name github-actions[bot]
           git config --global user.email 'github-actions@users.noreply.github.com'
       - name: Filename Formatter
-        run: |
-          wget https://raw.githubusercontent.com/TheAlgorithms/scripts/main/filename_formatter.sh
-          chmod +x filename_formatter.sh
-          ./filename_formatter.sh . .c,.h
+        uses: TheAlgorithms/scripts/formatter@main
+        with:
+          filetypes: .c,.h
```

Comment:
Shouldn't we specify the working directory, just in case?

Reply:
We can 😀 the reason I'm not worried about it is because of the default value set for the working_directory input in TheAlgorithms/scripts/formatter/action.yml